tox.ini

```diff
@@ -26,7 +28,7 @@ pytest_command =
 description = black checks [Mandatory]
 skip_install = true
 deps =
-    black==19.10b0
+    black==21.4b0
```

Comment:
Suggestion - separate PR or at least separate commit for bumping Black version - one can be lost in all those format changes which might not be relevant to the topic "Add support for Python3.9"

Reply:
Roger :+1:

Reply:
Addressed